src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form action="/search"
+    method="get"
+    class="form"
+    data-qa="big">
```

Comment:
Need to fix code style everywhere
```suggestion
    <form
      action="/search"
      method="get"
      class="form"
      data-qa="big"
    >
```